### PR TITLE
Toolchain fix for older python version

### PR DESCRIPTION
**Pull request by wilfonba**
This fixes the problem that I had building on wingtip for python 3.8.10 thanks to @henryleberre

### CODE REVIEW

File: toolchain/mfc/bench.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import os, sys, uuid, subprocess, dataclasses
 
-import rich.table
+import rich.table, typing
```
Comment (henryleberre) [inline toolchain/mfc/bench.py]: Nitpick but `typing` should be imported on the first line with the other Python standard library dependencies.

**Review by henryleberre — CHANGES_REQUESTED** [review]

File: toolchain/mfc/bench.py (hunk 1 above)
Comment (wilfonba) [reply toolchain/mfc/bench.py]: Done

**Comment by wilfonba** [comment]
No idea why this change messed with the benchmark time

**Comment by sbryngelson** [comment]
@henryleberre can merge this whenever it is done

**Comment by henryleberre** [comment]
Thanks @wilfonba. The increase in benchmark time is due to:
- Only running a few time steps
- Storing benchmark numbers as integers (seconds).

I believe @sbryngelson will add more cases.

**Comment by sbryngelson** [comment]
@wilfonba @henryleberre yeah it's in my queue

**Pull request merged**